NuGet.Config

```diff
@@ -28,5 +28,6 @@
     <add key="dotnet.myget.org roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
     <add key="myget.org vs-editor" value="https://myget.org/F/vs-editor/api/v3/index.json" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="vside.myget.org vssdk" value="https://vside.myget.org/F/vssdk/api/v3/index.json" />
```

Comment:
I see Shell.Framework 26606 has been released on nuget.org, so yes, I think you can remove this line.